source/BoardingPanel.cpp

```diff
@@ -228,22 +229,13 @@ bool BoardingPanel::KeyDown(SDL_Keycode key, Uint16 mod, const Command &command,
 		// When closing the panel, mark the player dead if their ship was captured.
 		if(playerDied)
 			player.Die();
-		// Handle any death benefits or profit shares that are owed.
-		if(deathBenefits)
-		{
-			Messages::Add(("You must pay " + Format::Number(deathBenefits)
-				+ " credits in death benefits for the ")
-				+ ((casualties > 1) ? "families of your dead crew members."
-					: "family of your dead crew member."));
-			player.Accounts().AddDeathBenefits(deathBenefits);
-		}
-		GetUI()->Pop(this);
+		// Handle any profit shares that are owed.
 		if(profitShares)
 		{
 			Messages::Add(("You must pay " + Format::Number(profitShares)
 				+ " credits in profit shares to the ")
 				+ (((you->Crew() - 1) > 1) ? "crew members who took part in this boarding action."
-					: "crew member who took part in this action."));
+					: "crew member who took part in this boarding action."));
```

Comment:
I expect we're paying profit shares to the whole fleet, so perhaps we could just say "your crew"?

Reply:
The profit sharing should absolutely take the whole fleet into account. The fleet enabled the player to board that ship in the first place by protecting the flagship, risking their lives in the process.

All escorts deserve a share of the profits.

Reply:
It's worth noting that one of the primary design goals of the profit sharing system is to make it less attractive for the player to have an overly large and powerful fleet. Smaller player fleets mean more risky combat encounters, which in turn makes for more exciting gameplay.

Reply:
Hmm.  You're right that all escorts contributed to making the boarding action possible.... but at the same time, the crewmembers of the flagship undertake, ignoring the meta of the player's ship is always going to survive, a much greater risk.  Not only do they have to sit in a capture barge that might be more vulnerable to attack while the victim is disabled, but the chances of someone dying during boarding are much greater.  It seems to me they ought to be paid more, corresponding to their greater risk.

Well, unless you're the sort of player who goes around with Sparrow fleets, then the fleet is probably taking a similar risk.